### PR TITLE
[DOCS] add Local Python Install to docs

### PR DESCRIPTION
**Pull request by petermoyer**
Also add 3 new components

Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE],  [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).

Changes proposed in this pull request:
- Add three new components to install
- Add local python install
- Add Spark EMR template
- Update sidebar.js (fixes typo and adds emoji)

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

### Definition of Done
Please delete options that are not relevant.

- [ X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [X ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [ X] I have commented my code, particularly in hard-to-understand areas
- [ X] I have made corresponding changes to the documentation
- [ ] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [ X] I have run any local integration tests and made sure that nothing is broken.

Thank you for submitting!

### CODE REVIEW

**Comment by petermoyer** [comment]
> Also add 3 new components
> 
> Please annotate your PR title to describe what the PR does, then give a brief bulleted description of your PR below. PR titles should begin with [BUGFIX], [FEATURE], [DOCS], or [MAINTENANCE]. If a new feature introduces breaking changes for the Great Expectations API or configuration files, please also add [BREAKING]. You can read about the tags in our [contributor checklist](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html).
> 
> Changes proposed in this pull request:
> 
> * Add three new components to install
> * Add local python install
> * Add Spark EMR template
> * Update sidebar.js (fixes typo and adds emoji)
> 
> After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.
> 
> ### Definition of Done
> Please delete options that are not relevant.
> 
> * [ X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
> * [X ] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
> * [ X] I have commented my code, particularly in hard-to-understand areas
> * [ X] I have made corresponding changes to the documentation
> * [ ]  I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
> * [ X] I have run any local integration tests and made sure that nothing is broken.
> 
> Thank you for submitting!

**Comment by petermoyer** [comment]
@cla-bot check

File: docs/guides/setup/installing/local.md [hunk 1]
```diff
@@ -1,5 +1,89 @@
 ---
-title: local python
+title: Installing in great_expectations in Local Python3
```
Comment (Aylr) [inline docs/guides/setup/installing/local.md]: Please make titles consistent with the specific language in the design document: https://github.com/superconductive/design/blob/main/docs/20210421_knoxpod_golden_paths_mental_map/toc.md#installation-this-still-needs-to-be-fleshed-out-and-finalized

File: docs/guides/setup/installing/spark-emr.md [hunk 2]
```diff
@@ -1,6 +1,48 @@
 ---
-title: Installing great_expectations in Spark EMR 
+title: Installing great_expectations in Spark EMR cluster
```
Comment (Aylr) [inline docs/guides/setup/installing/spark-emr.md]: Please make titles consistent with the specific language in the design document: https://github.com/superconductive/design/blob/main/docs/20210421_knoxpod_golden_paths_mental_map/toc.md#installation-this-still-needs-to-be-fleshed-out-and-finalized

File: docs/guides/setup/components/install_prereq.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React from 'react'
```
Comment (Aylr) [inline docs/guides/setup/components/install_prereq.jsx:1]: Rather than duplicating this component, perhaps we can move the existing one out to a more common place so it can be shared. We may want to factor out `defaultPrerequisiteItems` so it is more general and reusable. Happy to pair if you'd like!

File: sidebars.js [hunk 4]
```diff
@@ -32,7 +32,7 @@ module.exports = {
           items: [
             {
               type: 'category',
-              label: 'Installing',
+              label: '🧰 Installing',
```
Comment (Aylr) [inline sidebars.js]: `Installation` per the design spec. Thanks!

**Review by Aylr — CHANGES_REQUESTED** [review]
Awesome work. A few change requests.

File: docs/guides/setup/installing/local.md [hunk 5]
```diff
@@ -1,5 +1,89 @@
...
+```console
+python --version
+# or if multiple versions of python installed
+python3 --version
+python3 -m pip --version
+```
+
+### 2. Install Great Expectations
+```console
+pip install great_expectations
```
Comment (Shinnnyshinshin) [inline docs/guides/setup/installing/local.md:31]: Regarding installation, running `great_expectatsions --v3-api init` seems out of scope (and would be handled by a different doc). 

would we be able to have Step 2 be the following 

```
pip install great_expectations

great_expectations --version
```

And then have a link to 

`docs/guides/setup/configuring-data-contexts/how-to-create-a-new-data-context-with-the-cli.md?`

- this guide should cover the init and walking through the CLI steps.

File: docs/guides/setup/installing/local.md [hunk 6]
```diff
@@ -1,5 +1,89 @@
...
+
+Please see our documentation for more configuration options!
+
+```
+
+<Congratulations />
+
+## Additional Notes
+
+To view the full scripts used in this page, see them on GitHub:
```
Comment (Shinnnyshinshin) [inline docs/guides/setup/installing/local.md]: This can be removed

File: docs/guides/setup/installing/spark-emr.md [hunk 7]
```diff
@@ -1,6 +1,48 @@
...
+
+<Congratulations />
+
+## Additional Notes
+
+To view the full scripts used in this page, see them on GitHub:
+
+## Next Steps
+
+<NextSteps />
```
Comment (Shinnnyshinshin) [inline docs/guides/setup/installing/spark-emr.md:48]: I know that this was ported over from our previous docs, but in the way that we are writing the new docs, I'd like to request the following changes: 

Step 1:
- please include some additional information on where this command should be run:
`sc.install_pypi_package("great_expectations")`
-  Is it in a Notebook? is there a separate console?
- Also, to have parity with the local installation guide (`local.md`), it would be nice to have a command like `great_expectations --version` (or an EMR equivalent) to checking whether `great_expectations` was installed correctly. 

`Step 2` would become `Next Steps` and would say something like: 

In an EMR Spark Notebook, you will not have direct access to the CLI. Therefore you will need to create a Data Context in memory. To find out how please look at the document on  [How to instantiate a Data Context without a yml file](/docs/guides/setup/configuring-data-contexts/how-to-instantiate-a-data-context-without-a-yml-file)

File: sidebars.js [hunk 8]
```diff
@@ -281,7 +281,7 @@ module.exports = {
       items: [
         {
           type: 'category',
-          label: 'Contributingt',
+          label: 'Contributing',
           items: [
```
Comment (Shinnnyshinshin) [inline sidebars.js:285]: Thank you for catching this :)

**Review by Shinnnyshinshin — DISMISSED** [review]
Thank you very very much for these guides.  A few changes from me as well

File: docs/guides/setup/installing/spark-emr.md (hunk 7 above)
Comment (Shinnnyshinshin) [reply docs/guides/setup/installing/spark-emr.md:48]: The link will eventually move to a `Deployment Patterns` document called `How to instantiate a Data Context on an EMR Spark cluster` but it's not in the current branch (is being included by #2915)

File: docs/guides/setup/components/install_prereq.jsx (hunk 3 above)
Comment (petermoyer) [reply docs/guides/setup/components/install_prereq.jsx:1]: Will be handled in the next PR as it requires more changes

File: sidebars.js (hunk 4 above)
Comment (Aylr) [reply sidebars.js]: Fixed

File: docs/guides/setup/installing/local.md (hunk 1 above)
Comment (petermoyer) [reply docs/guides/setup/installing/local.md]: Done

File: docs/guides/setup/installing/local.md (hunk 5 above)
Comment (petermoyer) [reply docs/guides/setup/installing/local.md:31]: Done

File: docs/guides/setup/installing/spark-emr.md (hunk 2 above)
Comment (petermoyer) [reply docs/guides/setup/installing/spark-emr.md]: changed

File: docs/guides/setup/installing/spark-emr.md (hunk 7 above)
Comment (petermoyer) [reply docs/guides/setup/installing/spark-emr.md:48]: Will be handled in a future PR

**Review by Aylr — APPROVED** [review]
